packages/cms-data-sync/test/news/news.data-migration.test.ts

```diff
@@ -246,7 +246,8 @@ describe('migrateNews', () => {
       await migrateNews();
 
       expect(console.log).toHaveBeenCalledWith(
-        'There is a problem converting rich text from entry news-1',
+        '\x1b[31m',
```

Comment:
we can call this `RED_COLOR` now and replace the code by the variable (RED_COLOR/GREEN_COLOR/etc) in the other tests, right? :)

Reply:
imo correct way to do this would be to mock `logger` instead of `console.log` as we shouldn't be testing the underlying implementation of `logger` at this level.